### ISSUE TITLE
Changes are slow to sroll

### ISSUE THREAD

**Issue by McToel** [desktop/desktop#8968]
### Describe the feature or problem you’d like to solve

I am having the problem that I have way more changes sometimes than you should have. And then it's hard to get to the last or middle changes, because there is no scrolling bar at the side.

### Proposed solution

Add scrolling bar to the changes

**Comment by tierninho** [comment]
@McToel Thanks for the suggestion. 

We do have a scroll bar, however it depends on the number of files and size of your window. In summary, it will show if the files go past the area in sight. Alternately, you can always use the up/down keys to scroll. 

![Screen Shot 2020-01-23 at 11 06 17 AM](https://user-images.githubusercontent.com/14828183/73024031-cbce2c80-3dd0-11ea-8d4f-c6eaccab1f83.png)

Hope this helps!

**Comment by McToel** [comment]
Ok, I have to admit you are right, but I am working with something like 32 000 files and then the scroll bar is basically unusable: 
![grafik](https://user-images.githubusercontent.com/36071676/73028519-fc947d00-3e35-11ea-8cde-6e50ee7c2a05.png)

It would probably be right suggesting to use fewer files, but it would not harm anyone if GitHub Desktop properly supported many files better.

**Comment by tierninho** [comment]
Thanks for the screenshot! The scroll bar we use is the default scroll bar for native OSX apps, but it is rendered useless based on your situation. I agree, a custom UI could potentially resolve this. 

@billygriffin and @ampinsk for thoughts as this would like affect other parts of the app that use a scroll bar.

**Comment by niik** [comment]
Just trying to clean up the backlog slightly. Since this issue was opened we've introduced filtering in the changes list which should sufficiently address this issue. Going to close this out
